src/Equinox.MessageDb/MessageDb.fs

```diff
@@ -180,85 +67,62 @@ module Read =
     let inline len (bytes: EventBody) = bytes.Length
     let private resolvedEventLen (x: ITimelineEvent<EventBody>) = len x.Data + len x.Meta
     let private resolvedEventBytes events = events |> Array.sumBy resolvedEventLen
-    let private loggedReadSlice reader streamName batchSize requiresLeader startPos batchIndex (log: ILogger) ct: Task<_> = task {
+    let private loggedReadSlice reader streamName batchSize requiresLeader startPos batchIndex ct: Task<_> = task {
         let act = Activity.Current
         let! t, slice = readSliceAsync reader streamName batchSize startPos requiresLeader |> Stopwatch.time ct
```

Comment:
`t` is being dropped to the floor - surely the RTT for the loads is of interest? I personally am interested in slice roundtrips in addition to the overall load+fold time

Reply:
I'm banking on Npgsql spans for those timings